code/datums/keybinding/movement.dm

```diff
@@ -0,0 +1,27 @@
+/datum/keybinding/movement
+    category = CATEGORY_MOVEMENT
+    weight = WEIGHT_HIGHEST
+
+/datum/keybinding/movement/north
+	key = "W"
+	name = "North"
+	full_name = "Move Up"
+	description = "Moves your character north"
```

Comment:
The name says "up" and the description says "north". LITERALLY UNPLAYABLE

Reply:
If you don't know what north means you'll definitely know what up is